sycl/tools/sycl-ls/sycl-ls.cpp

```diff
@@ -126,6 +127,23 @@ std::array<int, 2> GetNumberOfSubAndSubSubDevices(const device &Device) {
   return {NumSubDevices, NumSubDevices * NumSubSubDevices};
 }
 
+/// Standard formatting of UUID numbers into a string
+/// e.g. "f81d4fae-7dec-11d0-a765-00a0c91e6bf6"
+std::string formatUUID(detail::uuid_type UUID) {
+  std::ostringstream oss;
```

Comment:
I'd recommend `oss.imbue(std::locale::classic());` to avoid localization side-effects.

Reply:
Done in https://github.com/intel/llvm/pull/18561/commits/ed7c57f84ed59155c99555086133d607ebd6e275